src/g_canvas.c

```diff
@@ -687,6 +693,11 @@ void canvas_map(t_canvas *x, t_floatarg f)
             if (x->gl_isgraph && x->gl_goprect)
                 canvas_drawredrect(x, 1);
             sys_vgui("pdtk_canvas_getscroll .x%lx.c\n", x);
+#ifdef HAVE_KEYBOARDNAV
+            t_kbdnav *kbdnav = canvas_get_kbdnav(x);
```

Comment:
make sure to conform to C89 (sic!) and declare your variables at the beginning of the code-block.

Reply:
okay!